clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -2313,6 +2313,20 @@ TEST(TransferTest, AssignmentOperatorWithInitAndInheritance) {
          ASTContext &ASTCtx) {});
 }
 
+TEST(TransferTest, InitListExprAsXValue) {
+  // This is a crash repro.
+  std::string Code = R"(
+    void target() {
+      auto&& test{false};
```

Comment:
```suggestion
      bool&& test{false};
```

Same number of characters, but more explicit.